client/app/scripts/liveblog-edit/views/post.html

```diff
@@ -7,7 +7,7 @@
         </div>
         <div class="lb-post__content">
             <div class="lb-post__header">
-                <span class="name">{{ ::post.original_creator_name }}</span> | <time>{{ post._updated | reldate }}</time>
```

Comment:
Why removing the one time binding? The `original_creator_name` shouldn't change I guess.